src/components/Entry.vue

```diff
@@ -110,6 +110,9 @@ export default {
       isLoading: false,
       invalidInputMessage: null, // message to show when the input being entered does not match the ID format,
       userType: "", // differentiates between different kinds of users
+      loadingSpinner: new URL('../assets/images/loading_spinner.svg', import.meta.url).href,
+      deleteSvg: new URL('../assets/images/remove_circle.svg', import.meta.url).href,
+      addSvg: new URL('../assets/images/add_circle.svg', import.meta.url).href
```

Comment:
There's a better way to do this, which I also encountered in a project I just set up.

You can use glob imports provided by vite to setup asset loading in one file and use it everywhere. https://vitejs.dev/guide/features.html#glob-import

Create a utility `.js` file which will do the asset handling. For example, `assets.js`
This is how the contents of the file might look like
```js
// inside assets.js

export default function useAssets() {
  // reference https://github.com/vitejs/vite/issues/1265#issuecomment-885948466
  const allJpegs = import.meta.globEager('/src/assets/*.jpeg')
  const allPngs = import.meta.globEager('/src/assets/*.png')
  const allSvgs = import.meta.globEager('/src/assets/*.svg')
  
return {
    loadingSpinnerIcon: jpegs['/src/assets/loading_spinner.svg'].default,
    deleteIcon: jpegs['/src/assets/remove_circle.svg'].default,
    .
    .
    .
  }
}
```

And then, in whatever file you want to use these, you can just do this
```js
// inside Entry.vue

import useAssets from '../../assets.js'

const assets = useAssets()
// then you can directly use `assets.loadingSpinner` and that will contain your SVG
```